test/integration/test_ibm_provider.py

```diff
@@ -117,6 +118,25 @@ def test_get_hgps_no_filter(self):
         hgps = self.provider._get_hgps()
         self.assertIn(self.provider.backend._default_hgp, hgps)
 
+    def test_active_account_instance(self):
+        """Test active_account returns correct instance."""
+        hgp = self.provider._get_hgp()
+        provider = IBMProvider(
+            token=self.dependencies.token,
+            url=self.dependencies.url,
+            instance=hgp.name,
+        )
+        self.assertEqual(hgp.name, provider.active_account()["instance"])
+
+    def test_active_account_instance_with_save_account(self):
+        """Test active_account returns correct instance with save_account."""
+        hgp = self.provider._get_hgp()
+        IBMProvider.save_account(
+            token=self.dependencies.token, instance=hgp.name, overwrite=True
+        )
```

Comment:
It would better not to save account on CI. Using something like a temporary account config file might help like here https://github.com/Qiskit/qiskit-ibm-provider/blob/main/test/unit/test_account.py#L174